prototype/howwework/project-entries.js

```diff
@@ -0,0 +1,145 @@
+window.entries = []
+
+window.entries.push({
+  title: 'Mind of my Own',
+  bannerimage: 'howwework/ourwork-momobanner.jpg',
+  image: 'howwework/ourwork-momo.jpg',
+  description: `We work with Mind of My Own to help young people participate fully in their care and make it easier to speak up anytime they want. We’ve built apps that make this happen,
+    leading on many aspects of the process.`,
+  problem: `Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.`,
+  solution: `Consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat`,
+  destination: 'https://www.mindofmyown.org.uk',
+})
+
+window.entries.push({
+  title: 'Alexander Rose',
```

Comment:
noooooo 😝

Reply:
needs to be changed to Alexandra Rose 😆 👍

Reply:
This content is changing anyway, https://github.com/neontribe/www/issues/98